addons/binding/org.openhab.binding.windcentrale/ESH-INF/thing/millThing.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="windcentrale" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="mill">
+		<label>WindCentrale Wind Mill</label>
+
+		<channels>
+			<channel id="windSpeed" typeId="windSpeedType" />
+			<channel id="windDirection" typeId="windDirectionType" />
+			<channel id="powerAbsTot" typeId="powerKType">
+				<label>Total Power</label>
+			</channel>
+			<channel id="powerAbsWd" typeId="powerType">
+				<label>WD Power</label>
+			</channel>
+			<channel id="powerRel" typeId="powerRelType">
+			</channel>
+
+			<channel id="kwh" typeId="energyType">
+			</channel>
+			<channel id="kwhForecast" typeId="energyType">
+				<label>Energy Forecast</label>
+			</channel>
+			<channel id="runPercentage" typeId="powerRelType">
+				<label>Run Percentage</label>
+				<description>Run Percentage this year</description>
+			</channel>
+			<channel id="timestamp" typeId="timestampType" />
+		</channels>
+
+		<properties>
+			<property name="vendor">WindCentrale</property>
+		</properties>
+
+		<config-description>
+			<parameter name="millId" type="integer" required="true">
+				<label>WindMill</label>
+				<options>
+					<option value="1">De Grote Geert</option>
+					<option value="2">De Jonge Held</option>
+					<option value="31">Het Rode Hert</option>
+					<option value="41">De Ranke Zwaan</option>
+					<option value="51">De Witte Juffer</option>
+					<option value="111">De Bonte Hen</option>
+					<option value="121">De Trouwe Wachter</option>
+					<option value="131">De Blauwe Reiger</option>
+					<option value="141">De Vier Winden</option>
+					<option value="191">De Boerenzwaluw</option>
+				</options>
+				<default>131</default>
+			</parameter>
+			<parameter name="wd" type="integer" required="false">
+				<label>Wind Parts</label>
+				<description># of wind parts ("winddelen")</description>
+				<default>1</default>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" min="1" max="9999" required="false">
+				<label>Refresh Interval</label>
+				<description>Refresh Interval for refreshing the data in s</description>
+				<default>30</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="windSpeedType">
```

Comment:
Too bad there is no convention for this. Bindings now all differentiate between:
* channelName
* channelname
* channel-name

The most common form I see used in bindings is camelCased.

Though I personally prefer lowercase. That's because I prefer the nicer looks with with the lowercase binding ID and thing ID. It also saves key strokes with typing, because you do not constantly need a shift modifier. Also you do not have to think about what is upper and lower case. But this is still all personal preference.